store/src/main/java/com/nytimes/android/external/store3/base/BasePersister.java

```diff
@@ -0,0 +1,8 @@
+package com.nytimes.android.external.store3.base;
+
+
+
+
+public interface BasePersister {
```

Comment:
Since we now have `RoomPersister` and `Persister` I made a marker interface for use with `StoreUtil` for things like checking stale policy or clearing the persister